nixos/modules/services/home-automation/zwave-js-ui.nix

```diff
@@ -0,0 +1,86 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    mkIf
+    mkEnableOption
+    mkOption
+    types
+    ;
+  cfg = config.services.zwave-js-ui;
+in
+{
+  options.services.zwave-js-ui = {
+    enable = mkEnableOption "zwave-js-ui";
+    serialPort = mkOption {
+      type = types.path;
+      description = ''
+        Serial port for the Z-Wave controller.
+
+        Used for permissions only; must be additionally set in the application
+      '';
+      example = "/dev/ttyUSB0";
+    };
+  };
+  config = mkIf cfg.enable {
+    assertions = [
+      {
+        assertion = !config.services.zwave-js.enable;
+        message = "zwave-js-ui conflicts with zwave-js";
```

Comment:
What if a user wants to try out one or the other in parallel? They could have a second network for example. I don't think this assertion is very convincing.